articles/zero-trust-attestation-with-fleet

```diff
@@ -0,0 +1,68 @@
+
+Fleet provides a desktop application called Fleet Desktop, which lives in the menubar. Fleet Desktop gives the user access to a Fleet webpage called "My Device," which, among others, features policy information. A user who has been denied access can visit this page to see the remediation instructions. This way, IT and security teams are partners of users in regaining access to company-owned resources. This self-remediation first approach reduces also the volume of help requests. 
+
+To learn more about installing Fleet Desktop, visit [here](https://fleetdm.com/docs/using-fleet/fleet-desktop).
+
+### Bring-your-own authentication provider
+One piece of the puzzle that you should bring, however, is your authentication provider. When your users request access, your authentication system should ping the Fleet API to check for policy compliance and make decisions about blocking based on the results. When presenting the access denied page, be sure to point users to Fleet Desktop's "My Device" page to help them find remediation instructions. 
+
+## Considerations for zero trust
+
+### Scalability and latency of the policy engine
+Since the request for policies will happen every time the user attempts to access company resources, this will generate a lot of requests. The Fleet policies endpoint has been optimized for quick pings. However, depending on the size of your organization, this will require tuning and upscaling the servers   to meet your organization's situation. 
+
+### Caching to reduce latency and load
+Instead of hitting the server every time an authentication request is made, one approach is to cache the policy results with a time to live (TTL). Say, every 5 minutes, the request would result in a recalculation of the policy results. This would reduce the load on the server and provide a low-latency response. 
+
+Caching, however, comes at two costs. First, it increases the amount of time it takes to recognize a policy failure (by at worst 5 minutes). During the TTL period, a device may have policy failures, but is still able to access your system, because the cache has not yet updated. Second, it would require your end user to wait at most 5 minutes after they've remediated the issue to regain access. The longer the TTL period, the less strain on your system, but the longer it takes for your zero trust system to reflect the facts on the ground.
+
+### Osquery policy recalculation period
+Furthermore, osquery itself has a delay between recalculating policies. By default, it will report new policy information every hour, but can be overridden using the `asdfasdasdf` setting. Depending on the strictness of your requirements, you may need to make this more frequent. However, note that a more frequent refresh period would cause a greater performance hit on the host. 
```

Comment:
Ah, this is what your question was about.

https://fleetdm.com/docs/deploying/configuration#osquery-policy-update-interval